client/src/styles/OrderField.scss

```diff
@@ -0,0 +1,13 @@
+.Resource__Order.field .form__field-holder {
+  input {
+      max-width: 6em;
+      min-width: 3em;
+      display: inline-block;
+  }
+
+  .form__field-description {
+    display: inline-block;
+    font-size: 1rem;
```

Comment:
Why do we have to specify a custom font size? Also something you probably expected; any reason there's no css variables that are suitable here?

Reply:
1rem is the default (base) font size, no matter the variable used to set it, which is why I opted for this approach. I'll have a hunt around for a variable, but feel like it's a time sink.

We must bring the font size up to the label size in order to match the design.
By default the description is smaller than the normal reading size.